image/gimp_brush.ksy

```diff
@@ -0,0 +1,32 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
+  license: CC0-1.0
+  endian: be
+doc-ref: https://gitlab.gnome.org/GNOME/gimp/-/blob/master/devel-docs/gbr.txt
```

Comment:
https://gitlab.gnome.org/GNOME/gimp/-/raw/master/devel-docs/gbr.txt

GitLab forcibly promotes JS in favour of GitLab inc. and their partners' commercial interests. Fortunately they have not yet restricted direct access to raw files and fortunately MarkDown or designed to be readable and writeable as plain text. If you want HTML markup, we can use GIMP GitHub mirror. https://github.com/GNOME/gimp/blob/master/devel-docs/gbr.txt Though GH is proprietary and belongs to M$,  it hehaves much more nicer.